app/models/physical_server_provision_request.rb

```diff
@@ -10,6 +10,14 @@ def my_role(_action = nil)
     'ems_operations'
   end
 
+  def my_queue_name
+    source.nil? ? super : source.queue_name_for_ems_operations
+  end
+
+  def source
+    @source ||= self.class.source_physical_server(source_id)
```

Comment:
@gmcculloug I couldn't use the same code as the firmware request because that used `options[:src_ids]` where this uses `#source_id`